### PR TITLE
Updating the script with a working version 

### PR DESCRIPTION
**Pull request by franciscopgomes**
Creating this PR in the correct repo (entra-docs) instead of azure-docs.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@cilwerner 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity-platform/jwt-claims-customization.md [hunk 1]
```diff
@@ -275,83 +275,43 @@ To run this script, you need:
...
-$fqdn="fourthcoffeetest.onmicrosoft.com" # this is used for the 'issued to' and 'issued by' field of the certificate
-$pwd="mypassword" # password for exporting the certificate private key
-$location="C:\\temp" # path to folder where both the pfx and cer file will be written to
+##########################################################
+# Replace the variables below with the appropriate values 
+
+$fqdn="yourDomainHere" # This is used for the 'issued to' and 'issued by' field of the certificate
+$pwd="password" # password for exporting the certificate private key
+$tenantId   = "franciscogom.es" # Replace with Tenant ID
+$appObjId = "384bd956-daaa-4ac3-b558-a1e5ceb15bd2" # Replace with the Object ID of the App Registration
```
Comment (cilwerner) [inline docs/identity-platform/jwt-claims-customization.md:284]: Is it the Application (client) id or the Object ID here? They are two different things. Is the tenant ID a GUID, or a domain?

Replacing GUIDs/values as they are sensitive info

```suggestion
$tenantId   = "aaaabbbb-0000-cccc-1111-dddd2222eeee" # Replace with your Tenant ID
$appObjId = "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" # Replace with the Object ID of the App Registration
```

File: docs/identity-platform/jwt-claims-customization.md [hunk 2]
```diff
@@ -275,83 +275,43 @@ To run this script, you need:
...
-[string[]] $Scopes = "https://graph.microsoft.com/.default"
 $pfxpath = $pfxFile # path to pfx file
 $cerpath = $cerFile # path to cer file
-$SPOID = "<service-principal-id>"
-$graphuri = "https://graph.microsoft.com/v1.0/serviceprincipals/$SPOID"
 $password = $pwd  # password for the pfx file
  
- 
-# choose the correct folder name for MSAL based on PowerShell version 5.1 (.Net) or PowerShell Core (.Net Core)
+# Check PowerShell version 5.1 (.Net) or PowerShell Core (.Net Core) and read the certificate file accordingly
```
Comment (cilwerner) [inline docs/identity-platform/jwt-claims-customization.md:305]:
```suggestion
# Check PowerShell version (minimum 5.1) (.Net) or PowerShell Core (.Net Core) and read the certificate file accordingly
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
A few minor changes and clarifications needed here.

**Comment by cilwerner** [comment]
@franciscopgomes, please fix the recommended suggestions. Thanks

**Comment by franciscopgomes** [comment]
> @franciscopgomes, please fix the recommended suggestions. Thanks

@cliwerner - the Object ID was actually a random guid generated for sample purposes (and yes, we need the Object ID of the App Reg and not the App ID here). Tenant ID will take either a verified domain or the Tenant ID GUID. I commited the suggested changes.

**Review by cilwerner — APPROVED** [review]
Looks good now @franciscopgomes.

**Comment by cilwerner** [comment]
#sign-off

**Pull request merged**